kikit/fab/pcbway.py

```diff
@@ -116,6 +126,19 @@ def bomToCsv(bomData, filename, nBoards, types):
                              solderType, notes])
             item_no += 1
 
+def isPositonUnused(bomData, position):
+    for references in bomData.items():
+        for item in references[1]:
+            # print("references - ", references, "  position[0] - ", position[0])
+            if item == position[0]:
+                return False
+    return True
+
+def ignoreUnusedPosition(posData, bomData):
+    for x in posData:
+        if isPositonUnused(bomData, x):
+            posData.remove(x)
+    return posData
```

Comment:
Do you need this? I this you only need to filter BOM on the line below - see JLC for example.